src/raft.rs

```diff
@@ -303,6 +303,14 @@ fn new_message(to: u64, field_type: MessageType, from: Option<u64>) -> Message {
     m
 }
 
+fn get_priority(m: &Message) -> i64 {
+    if m.priority != 0 {
+        m.priority
+    } else {
+        0_i64.saturating_add_unsigned(m.deprecated_priority)
```

Comment:
This is a nightly only API. You can do that within stable `try_from.unwrap_or(max)`

Reply:
Fixed, thanks!